models/stg_twilio.yml

```diff
@@ -456,7 +456,7 @@ models:
           - not_null
         
       - name: created_at
-        description: 	The date that this resource was created, given in RFC 2822 format.
+        description: The date that this resource was created, given in RFC 2822 format.
```

Comment:
Similarly, I needed to remove the whitespace in order for the package to run on my end.

Reply:
fixed